dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -62,14 +62,53 @@ static const columninfo RecycleBinColumns[] =
  * Recycle Bin folder
  */
 
-HRESULT CRecyclerExtractIcon_CreateInstance(LPCITEMIDLIST pidl, REFIID riid, LPVOID * ppvOut)
+BOOL WINAPI CBSearchRecycleBin(IN PVOID Context, IN HANDLE hDeletedFile);
+
+static PIDLRecycleStruct * _ILGetRecycleStruct(LPCITEMIDLIST pidl);
+
+typedef struct
```

Comment:
```suggestion
typedef struct _SEARCH_CONTEXT
```

Reply:
i haven't changed that, i only reaordered it, anyways i can add that change if it's needed

Reply:
Done